<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Partner_win</class>
 <widget class="QDialog" name="Partner_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>332</width>
    <height>527</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>333</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Мастер пол</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 232, 211);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>126</y>
     <width>315</width>
     <height>403</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_show_products">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_select_product">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_company_name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_total_amount"/>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_price_with_discount">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Цена со скидкой:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Цена товара:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Выбор материала</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Наименование компании</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_main_price">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Выбор товара</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Количетсво товара</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_form_order">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>44</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{     
background-color: #67BA80;  /* Цвет фона кнопки */    
color: white;  /* Цвет текста */     border-radius: 5px;  /* Скругленные углы */ 
}</string>
      </property>
      <property name="text">
       <string>Купить товар</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_products_table">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>44</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{     
background-color: #67BA80;  /* Цвет фона кнопки */    
color: white;  /* Цвет текста */     border-radius: 5px;  /* Скругленные углы */ 
}</string>
      </property>
      <property name="text">
       <string>Таблица товаров</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_products_table_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>44</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{     
background-color: #67BA80;  /* Цвет фона кнопки */    
color: white;  /* Цвет текста */     border-radius: 5px;  /* Скругленные углы */ 
}</string>
      </property>
      <property name="text">
       <string>Просмотр заказов</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>55</horstretch>
     <verstretch>55</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../task and images/Мастер пол.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit_p">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{     
background-color: #67BA80;  /* Цвет фона кнопки */    
color: white;  /* Цвет текста */     border-radius: 5px;  /* Скругленные углы */ 
}</string>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
